src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
```

Comment:
```suggestion
      {`My name is ${person.name}`}
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals